package-lock.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
what is this? Copilot, what have you done?!

Reply:
Copilot: 
The pull request #68 titled "Ensure power is a number throughout all code" updates the package-lock.json file along with changes in parsing/src/interfaces/Recipe.ts and parsing/src/processor.ts to ensure that power is treated as a number instead of a string. This likely involved updating dependencies or regenerating the package-lock.json file to match the updated project requirements. For more details, you can review the full PR [here](https://github.com/Maelstromeous/satisfactory-factories/pull/68).

Reply:
Not helpful Copilot...